components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -283,7 +287,7 @@ export function createDefaultIntroMessage(channel, centeredIntro, enableUserCrea
                             {(title) => (
                                 <i
                                     className='fa fa-user-plus'
-                                    title={title}
+                                    title={title as string}
```

Comment:
Why do we need this type casting here and in the change above? Can't we add the type to this lambda's parameter?

Reply:
You're right, I added the type to the lambda parameter and removed the cast.